Source/Core/Core/HW/DVDInterface.h

```diff
@@ -106,20 +106,16 @@ void DoState(PointerWrap& p);
 
 void RegisterMMIO(MMIO::Mapping* mmio, u32 base);
 
-// Direct disc access
+// Disc access (don't call GetVolume unless you know that IsDiscInside() == true)
 const DiscIO::IVolume& GetVolume();
 bool SetVolumeName(const std::string& disc_path);
 bool SetVolumeDirectory(const std::string& disc_path, bool is_wii,
                         const std::string& apploader_path = "", const std::string& DOL_path = "");
-bool VolumeIsValid();
-
-// Disc detection and swapping
-void SetDiscInside(bool _DiscInside);
 bool IsDiscInside();
 void ChangeDiscAsHost(const std::string& new_path);  // Can only be called by the host thread
 void ChangeDiscAsCPU(const std::string& new_path);   // Can only be called by the CPU thread
 
-// DVD Access Functions
+// Direct access to DI for IOS HLE (a bit different from how it actually works on hardware)
```

Comment:
The difference is basically that the real IOS accesses DI in the same way as PPC software would, whereas we just use C++ calls instead. The reasons are because it's simpler, and because it lets us skip first encrypting and then decrypting Wii disc content in some cases. Readers who are familiar with the Wii should already know that IOS can't just make a C++ call into DI. If there's anything more that should be added to the comment, I'm not sure how to word it.